zsh/.zshrc

```diff
@@ -257,6 +257,13 @@ if [[ -z "$LS_COLORS" ]]; then
   export LS_COLORS='di=1;34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
 fi
 
+if [[ $(_zqs-get-setting ssh-askpass-require true) == 'true' ]]; then
+  echo
+  echo "Setting SSH_ASKPASS_REQUIRE=never,\nthis will prompt for your ssh passphrase at the command line"
+  export SSH_ASKPASS_REQUIRE=never
+  echo
+fi
+
```

Comment:
I am not sure if this is right right place. I ran the file through a bash debugger and it was jumping all over the place, until it reached my breakpoint.

Reply:
That's a good place to put it.